packages/fxa-payments-server/src/store/utils.tsx

```diff
@@ -13,91 +13,6 @@ export const mapToObject = (
   return out;
 };
 
-type ErrorResponseBody = {
-  code?: string;
-  statusCode?: number;
-  errno?: number;
-  error?: string;
-  message?: string;
-  info?: string;
-};
-
-export class APIError extends Error {
-  body: ErrorResponseBody | null;
-  response: Response | null | undefined;
-  code: string | null;
-  statusCode: number | null;
-  errno: number | null;
-  error: string | null;
-
-  constructor(
-    body?: ErrorResponseBody,
-    response?: Response,
-    code?: string,
-    errno?: number,
-    error?: string,
-    statusCode?: number,
-    ...params: Array<any>
-  ) {
-    super(...params);
-    this.response = response;
-    this.body = body || null;
-    this.code = code || null;
-    this.statusCode = statusCode || null;
-    this.errno = errno || null;
-    this.error = error || null;
-
-    if (this.body) {
-      const { code, errno, error, message, statusCode } = this.body;
-      Object.assign(this, { code, errno, error, message, statusCode });
-    }
-  }
-}
-
-// TODO: Use a better type here
-interface APIFetchOptions {
-  [propName: string]: any;
-}
-
-export const apiFetch = async (
-  method: string,
-  accessToken: string,
-  path: string,
-  options: APIFetchOptions = {}
-) => {
-  const response = await fetch(path, {
-    mode: 'cors',
-    credentials: 'omit',
-    method,
-    ...options,
-    headers: {
-      'Content-Type': 'application/json',
-      Authorization: `Bearer ${accessToken}`,
-      ...(options.headers || {}),
-    },
-  });
-  if (response.status >= 400) {
-    let body = {};
-    try {
-      // Parse the body as JSON, but will fail if things have really gone wrong
-      body = await response.json();
-    } catch (_) {
-      // No-op
-    }
-    throw new APIError(body, response);
-  }
-  return response.json();
-};
-
-export const apiGet = (...args: [string, string, object?]) =>
-  apiFetch('GET', ...args);
-
-export const apiDelete = (...args: [string, string, object?]) =>
-  apiFetch('DELETE', ...args);
-
-export const apiPost = (accessToken: string, path: string, body: object) =>
-  apiFetch('POST', accessToken, path, { body: JSON.stringify(body) });
-
 export const setStatic = (newState: object) => (state: object): object => ({
```

Comment:
Jest has our branch test coverage @ 50% for `setFromPayLoad` (line 24) due to the return of `payload || defval` for `[name]`. I can write a test for this, but I don't see that we're using `setAsPayload`, `setFromPayload`, or `setFromPayLoadFn` anywhere. Can I remove these functions & their related tests?

Reply:
Yeah, if this is dead code now, feel free to remove it! I think it might have been used in the react-actions code that you refactored away